### PR TITLE
Project 8

### PR DESCRIPTION
**Pull request by matthewlang**
Adds project 8.

Major changes from my/Catie's previous version is the updated libraries.

This project burns CPU. I might consider upping the CPU limit in the config for this...

### CODE REVIEW

File: projects/project08/P8.ipynb [hunk 1]
```diff
@@ -0,0 +1,464 @@
...
+    "print(make_output_filename('lynx.ppm', 1)) # should print lynx_negate_red.ppm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Write your own tets here."
```
Comment (pkirlin) [inline projects/project08/P8.ipynb:201]: s/tets/tests

**Review by pkirlin — COMMENTED** [review]
looks good!  one typo.

**Comment by pkirlin** [comment]
can't attach ppm here, see slack.

You can say "Prof Kirlin is lame and has no pets, so here's his snowman, Blobby McPastaface."

File: projects/project08/P8.ipynb (hunk 1 above)
Comment (matthewlang) [reply projects/project08/P8.ipynb]: done, thanks!

**Comment by matthewlang** [comment]
Thanks! Merging!

**Pull request merged**